doc/Analytics.xml

```diff
@@ -3142,7 +3146,9 @@
   </tt:Source>
   <tt:Data>
     ...
-  </tt:Data>
+    <tt:SimpleItemDescription Name="ObjectId" Type="xs:integer"/>
```

Comment:
Many rules already have ObjectID of type StringList. Propose to either align name or drop.